esp/scm/WsDeploy.ecm

```diff
@@ -568,7 +568,7 @@ ESPresponse [exceptions_inline, encode(0)] GetSummaryResponse
     string XmlStr;
 };
 
-ESPservice [requiredaccess("NONE"), exceptions_inline("xslt/exceptions.xslt")] WsDeploy
+ESPservice [autofeatureauth("NONE"), exceptions_inline("xslt/exceptions.xslt")] WsDeploy
```

Comment:
@wangkx none of the methods in this service seem to require any feature level auth, is this correct?